packages/fiori/cypress/specs/SideNavigation.cy.tsx

```diff
@@ -308,6 +308,62 @@ describe("Side Navigation interaction", () => {
 		});
 	});
 
+	it("Tests link opening with mouse click", () => {
+		cy.mount(
+			<SideNavigation id="sideNav">
+				<SideNavigationItem id="item" text="1" />
+				<SideNavigationItem id="unselectableItemWithLink" text="external link" target="_blank" unselectable={true} href="#test"/>
+				<SideNavigationItem text="3">
+					<SideNavigationSubItem id="SubItemWithLink" text="external link" target="_blank" href="#test"/>
+				</SideNavigationItem>
+			</SideNavigation>
+		);
+
+		cy.url().should("not.include", "#test");
+		cy.get("#unselectableItemWithLink").shadow().find("a").invoke("removeAttr", "target");
+
+		cy.get("#unselectableItemWithLink").realClick();
+
+		cy.url().should("include", "#test");
+
+		// Remove #test from the URL
+		cy.window().then(win => {
+			win.history.pushState({}, "", win.location.pathname + win.location.search);
+		});
+
+		cy.url().should("not.include", "#test");
+	});
```

Comment:
1. The mounted SideNavigation contains much more than it is needed for testing
2. There is no need to set `target="_blank"` and then to remove it manually
3. You can just reset the history with `window.back()`

My suggestion for the test is:
```ts
it("Tests link opening with mouse click", () => {
		cy.mount(
			<SideNavigation id="sideNav">
				<SideNavigationItem id="unselectableItemWithLink" text="external link" unselectable={true} href="#test"/>
			</SideNavigation>
		);

		cy.url().should("not.include", "#test");

		cy.get("#unselectableItemWithLink").realClick();

		cy.url().should("include", "#test");

		// Remove #test from the URL
		cy.window().then(win => {
			win.history.back();
		});

		cy.url().should("not.include", "#test");
});
```